src/fu-self-test.c

```diff
@@ -4205,8 +4203,6 @@ fu_backend_usb_invalid_func(gconstpointer user_data)
 
 	/* check the fwupd DS20 descriptor was *not* parsed */
 	g_assert_false(fu_device_has_icon(device_tmp, "computer"));
-#else
-	g_test_skip("No GUsb support");
```

Comment:
this block should stay so that usb tests could be properly skipped when not compiled with usb